### DISCUSSION TITLE
1.18 support?

### DISCUSSION THREAD

**Discussion by NakkiiBrks** [PrismarineJS/mineflayer#2376] [Q&A] [answered]
When will support for 1.18 come out?

**Comment by amoraschi** [comment, marked as answer]
When it's completed

https://github.com/PrismarineJS/mineflayer/issues/2329
https://github.com/PrismarineJS/mineflayer/pull/2339

**Reply by NakkiiBrks** [reply]
Ok, thanks.

**Reply by AnushkDEV1** [reply]
please do it soon
